vllm/worker/worker.py

```diff
@@ -165,6 +165,8 @@ def init_cache_engine(self, cache_config: CacheConfig) -> None:
     def warm_up_model(self) -> None:
         if not self.model_config.enforce_eager:
             self.model_runner.capture_model(self.gpu_cache)
+        if self.cache_config.enable_prefix_caching:
+            self.model_runner.warmup_prefix_attn(self.gpu_cache)
```

Comment:
Is this called only in profiling? Or each inference?

Reply:
`warm_up_model` is not called on the hotpath